demo/test.c

```diff
@@ -87,6 +90,24 @@ static int test_trivial_stuff(void)
    mp_set_int(&c, 7uL);
    mp_exptmod(&a, &b, &c, &d);
 
+   mp_set_int(&a, 123456uL);
+   tmp = tmpfile();
+   if ((e = mp_fwrite(&a, 64, tmp)) != MP_OKAY) {
+      goto LBL_ERR;
+   }
+   rewind(tmp);
+   if ((e = mp_fread(&b, 64, tmp)) != MP_OKAY) {
```

Comment:
Even if these are trivial tests, could you move them to their own test_* functions please?

Reply:
fine like that?

Reply:
Not that it matters, but there are missing fcloses.

Reply:
Besides that :+1:

Reply:
better?